web/src/app/schedules/ScheduleOverrideCreateDialog.tsx

```diff
@@ -1,12 +1,26 @@
 import React, { useState } from 'react'
 import { gql, useMutation } from '@apollo/client'
-import p from 'prop-types'
 import FormDialog from '../dialogs/FormDialog'
 import { DateTime } from 'luxon'
 import ScheduleOverrideForm from './ScheduleOverrideForm'
 import { fieldErrors, nonFieldErrors } from '../util/errutil'
 import useOverrideNotices from './useOverrideNotices'
 
+interface ScheduleOverrideCreateDialogProps {
+  scheduleID: string
+  variant: 'add' | 'remove' | 'replace'
+  onClose?: () => void
+  removeUserReadOnly?: boolean
+  defaultValue?: DefaultValue
+}
+
+interface DefaultValue {
```

Comment:
it doesn't look like the two files that render this component pass a `defaultValue` props. I'm thinking we can delete this and its references